lib/libspl/include/umem.h

```diff
@@ -137,6 +137,21 @@ umem_free(const void *ptr, size_t size __maybe_unused)
 	free((void *)ptr);
 }
 
+/*
+ * umem_free_aligned was readded for supporting portability
```

Comment:
I think added is more appropriate. Saying readded here does not make sense to me.

Reply:
Done